dls-controls/malcolmjs
# Feature/url routing #84

## Description

Updated the url handling so the block ids are taken from the parts of the url path rather than the query. Also updated the main malcolm view so it displays the details about the child.
Add a github template for future pull requests (still experimenting to see if this is picked up automatically)

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- Run `npm start` and navigate to http://localhost:3000/gui/PANDA//PANDA:SEQ1 and observe both the parent and child panels being populated.

## Agile board tracking

connect to #84 
connect to #82
